web/src/i18n/locales/en.yaml

```diff
@@ -174,14 +174,14 @@
           subheader: "Include as much detail as necessary to explain the activity."
         hie:
           subheader: "Describe the activity in detail including images, diagrams, or other relevant information."
-          helpText: "The activity details must also include a description of three items: how the activity is linked to meaningful use; value proposition to providers, and sustainability plan describing how the activity will be sustained after HIE DDI funding is exhausted.	Additionally, include diagrams and descriptions of other HIE linkages as needed if they provide more context for how you'll complete the activity."
+          helpText: "The activity details must also include a description of three items: how the activity is linked to meaningful use; value proposition to providers, and sustainability plan describing how the activity will be sustained after HIE DDI funding is exhausted.	Additionally, include diagrams and descriptions of other HIE linkages as needed if they provide more context for how your state will complete the activity."
```

Comment:
Should this one be "...how the state will..."?  It looks like you've removed "your state" elsewhere, so just curious.

Reply:
I'll fix it.